.github/workflows/backlog-automation.yml

```diff
@@ -0,0 +1,18 @@
+name: Add all new issues & PRs to the Automata's board.
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types: opened
```

Comment:
Not sure about adding PRs in the New column. I would likely add a non draft and ready for review PRs in the In review column (as much). But ideally I'd suggest to rely solely on the issue (with the attached PR). Otherwise the board would be a bit overloaded and redundant IMO.

Reply:
As a backup... in Ventures implementation seems they do only add the issues. 

https://github.com/woocommerce/facebook-for-woocommerce/blob/develop/.github/workflows/backlog-automation.yml

Reply:
Seems like after discussion we are going to add PRs as well